### PR TITLE
Feature/workflow registration wizard

### PR DESCRIPTION
**Pull request by agduncan94**
https://docs.dockstore.org/en/feature-workflow-registration-wizard/getting-started/dockstore-workflows.html

Started on documentation for 1.8.0. This adds notes on the new whitelist wizard. I rewrote most of the page (minus the last few sections) so I recommend looking at the link above for the review.

https://github.com/dockstore/dockstore/issues/3115

### CODE REVIEW

File: docs/getting-started/dockstore-workflows.rst [hunk 1]
```diff
@@ -47,149 +48,125 @@ below:
...
-on GitHub than Docker images on Quay.io.
+Prepare Your Workflow for Dockstore
+-----------------------------------
+This tutorial does not go through the creation of a workflow and its
+registration to GitHub, Bitbucket or GitLab. It assumes that you already
+have a repository which contains a workflow and are now trying to register
+it in Dockstore. There are some ways to make the registration process more
+seamless.
+
+- For your primary workflow descriptor, use the filename ``Dockstore.<ext>``
```
Comment (coverbeck) [inline docs/getting-started/dockstore-workflows.rst]: Maybe give examples, like Dockstore.wdl and Dockstore.cwl. Also, is that the convention for Nextflow?

File: docs/getting-started/dockstore-workflows.rst [hunk 2]
```diff
@@ -47,149 +48,125 @@ below:
...
+
+By default, Dockstore will search the root of your repository for workflow
+related files. Following the above tips will help streamline the registration
+process, though you can still register workflows with non-standard format.
 
 Register Your Workflow in Dockstore
 -----------------------------------
+There are a variety of ways to get your workflows into Dockstore. There are two
+types of quick registration and one type of manual registration. These quick
+methods create workflow stubs, which are simply workflows that have not yet
```
Comment (coverbeck) [inline docs/getting-started/dockstore-workflows.rst]: I thought you got rid of stubs?

File: docs/getting-started/dockstore-workflows.rst [hunk 3]
```diff
@@ -47,149 +48,125 @@ below:
...
+not allow you to create workflows with workflow names.
+To do that you must do manual registration, which is described later.
 
-Manual Registration of Workflows
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Refresh All
+^^^^^^^^^^^
+Refresh all will look at all of your third-party accounts and do the following:
+
+- Create stub workflows for all Git repositories which do not exist on
```
Comment (coverbeck) [inline docs/getting-started/dockstore-workflows.rst]: Same comment/question regarding stubs

**Comment by agduncan94** [comment]
@coverbeck So stubs have not been fully removed since we have not removed refresh all. I think the plan is to get rid of them in 1.9.0. So instead of having a stub it would just be a workflow with no versions yet, since in this new way of doing things a user should only be adding workflows they bless.

File: docs/getting-started/dockstore-workflows.rst [hunk 4]
```diff
@@ -47,149 +48,125 @@ below:
...
+  Dockstore
+- Refersh all workflows that have been converted from stub to full workflows
```
Comment (garyluu) [inline docs/getting-started/dockstore-workflows.rst]: Literally unread-able

Comment (agduncan94) [reply docs/getting-started/dockstore-workflows.rst]: You don't like refersh?

**Comment by garyluu** [comment]
https://galaxyproject.org/ cert expired :(

File: docs/getting-started/dockstore-workflows.rst (hunk 2 above)
Comment (agduncan94) [reply docs/getting-started/dockstore-workflows.rst]: Replied to this in a general comment below.

File: docs/getting-started/dockstore-workflows.rst (hunk 3 above)
Comment (agduncan94) [reply docs/getting-started/dockstore-workflows.rst]: Replied to this in a general comment below.

**Pull request merged**